docs/data/base/components/switch/UseSwitchesBasic.tsx

```diff
@@ -36,22 +36,33 @@ const blue = {
 };
 
 const grey = {
-  400: '#BFC7CF',
-  500: '#AAB4BE',
+  50: '#F3F6F9',
+  100: '#E7EBF0',
+  200: '#E0E3E7',
+  300: '#CDD2D7',
+  400: '#B2BAC2',
+  500: '#A0AAB4',
   600: '#6F7E8C',
+  700: '#3E5060',
+  800: '#2D3843',
+  900: '#1A2027',
 };
 
 const BasicSwitchRoot = styled('span')(
   ({ theme }) => `
   font-size: 0;
   position: relative;
   display: inline-block;
-  width: 40px;
+  width: 48px;
   height: 24px;
   margin: 10px;
   cursor: pointer;
-  border-radius: 16px;
-  background: ${theme.palette.mode === 'dark' ? grey[600] : grey[400]};
+  background: ${theme.palette.mode === 'dark' ? grey[800] : grey[50]};
+  border: 1px solid ${theme.palette.mode === 'dark' ? grey[700] : grey[300]};
+  border-radius: 24px;
+  box-shadow: inset 0px 1px 1px ${
+    theme.palette.mode === 'dark' ? 'rgba(0, 0, 0, 0.25)' : 'rgba(0, 0, 0, 0.1)'
```

Comment:
```suggestion
    theme.palette.mode === 'dark' ? 'rgba(0, 0, 0, 0.5)' : 'rgba(0, 0, 0, 0.05)'
```

Here, as well! And slightly more aggressive in dark mode.